### PR TITLE
^Changelogs Required Above This Point^

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Changelogs are now required for all *feature* PRs above this point.

Pure fix PRs do not require a changelog (you can make one if you want...), but feature PRs will.

If you're curious how to set up/utilize the changelog, please refer to the readme: https://github.com/ParadiseSS13/Paradise/blob/master/html/changelogs/__CHANGELOG_README.txt and see the example: https://github.com/ParadiseSS13/Paradise/blob/master/html/changelogs/example.yml

Another guide: https://tgstation13.org/wiki/Guide_to_Changelogs

### CODE REVIEW

**Comment by TheDZD** [comment]
As of the current moment, :cl: has not yet been implemented for changelogs, you do need to create a .yml file and follow the instructions in the changelog readme. PR inline changelogging should be implemented sometime in the near future.

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -1,1439 +1 @@
...
-
-/mob/proc/can_unbuckle(mob/user)
-	return 1
-
-
-//Can the mob see reagents inside of containers?
-/mob/proc/can_see_reagents()
-	return 0
\ No newline at end of file
+MOBS SUCKS
```
Comment (Crazylemon64) [inline code/modules/mob/mob.dm]: word

Comment (Fox-McCloud) [inline code/modules/mob/mob.dm]: Literally 99.9% of SS13's performance problems.

Comment (Crazylemon64) [inline code/modules/mob/mob.dm]:
```
/mob/proc/New()
   gib()
```
Fix:
* Game is 100% better

**Comment by tigercat2000** [comment]
please god don't leave a PR open forever about this
i'll yell at you if this isn't closed in a reasonable amount of time, like a month, @Fox-McCloud

File: code/modules/mob/mob.dm (hunk 1 above)
Comment (Tastyfish) [inline code/modules/mob/mob.dm]:
```dm
/datum/New()
    qdel(src)
```
should be more efficient.

Comment (Crazylemon64) [inline code/modules/mob/mob.dm]: Less `!!PIZAZZ!!`, though

Comment (Tastyfish) [inline code/modules/mob/mob.dm]: Yes

Comment (CrAzYPiLoT-SS13) [inline code/modules/mob/mob.dm]: I've been wanting this for half a year now. Thanks Fox!

Comment (Fox-McCloud) [inline code/modules/mob/mob.dm]: Please thank @TheDZD He put all the hard work into getting this up and running.

**Pull request closed without merging**